fix(curriculum): external lu decomposition link in CIP (#46286)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/lu-decomposition.md

# --description--

> Every square matrix $A$ can be decomposed into a product of a lower triangular matrix $L$ and a upper triangular matrix $U$, as described in LU decomposition.

Reviewer: ```suggestion
Every square matrix $A$ can be decomposed into a product of a lower triangular matrix $L$ and a upper triangular matrix $U$. This is known as the LU decomposition.
```
